### PR TITLE
blood cult change 1, talisman changes

### PR DESCRIPTION
**Pull request by ghost**
changes the talisman of disguising to have the correct description for hiding runes

adds a talisman of barrier making, currently unavailable, to cultists, which creates an 80 health sanguine barrier when used

stunpapering a ratvar cultist hurts them more

talisman of horrors is now suitably silent and thus not awful compared to anything else

talisman of construction will-convert walls into cult walls, deconstruct cult walls into cult metal, and repair constructs as well

soulstone making cost increased to 25 rglass

adds a proc for talisman consumption that fixes a couple things

cuffbinding a ratvar cultist will hurt them using a talisman of shackling

teleport talisman has 3 uses

talisman construction has 4

#### Changelog

:cl:
rscadd: Construction talismans can now convert regular walls into cult walls, and deconstruct cult walls, and repair constructs.
rscadd: Construction talismans have 4 uses, teleport talismans have 3 uses.
rscadd: Adds the talisman of barrier formation. 
rscadd: Stunpapering a ratvar cultist will properly injure them extra.
rscadd: Soulstone making with rglass now costs 25 instead of 10 rglass.
rscadd: Bugfix-the talisman of disguising now accurately hides runes with the correct description.

/:cl:

### CODE REVIEW

**Comment by Super3222** [comment]
fuck you i was planning on making bloodcult content

**Comment by AndySok** [comment]
> teleport talisman has 3 uses

oh yay we're upgrading an OP talisman.

Kewl

File: code/game/gamemodes/cult/talisman.dm [hunk 1]
```diff
@@ -303,7 +328,8 @@
 	cultist_name = "Talisman of Horrors"
 	cultist_desc = "A talisman that will break the mind of the victim with nightmarish hallucinations."
 	color = "#ffb366" // light orange
-	invocation = "Lo'Nab Na'Dm!"
+	item_state = "null" //invisible inhand so cultists can smack a bitch with it silently
```
Comment (Time-Green) [inline code/game/gamemodes/cult/talisman.dm]: Just put null, not "null"

Comment (ShadowDeath6) [reply code/game/gamemodes/cult/talisman.dm]: "null!"

File: code/game/gamemodes/cult/talisman.dm [hunk 2]
```diff
@@ -221,6 +222,27 @@
 	user.visible_message("<span class='warning'>[user]'s hand flashes a bright blue!</span>", \
 						 "<span class='cultitalic'>You speak the words of the talisman, emitting an EMP blast.</span>")
 	empulse(src, 4, 8)
+	
+//Rite of Protection: Creates a sanguine barrier when used, good for OH FUCK HELP SEC AFTER ME
+/obj/item/weapon/paper/talisman/barrier
+	cultist_name = "Talisman of Barrier Formation
```
Comment (Time-Green) [inline code/game/gamemodes/cult/talisman.dm]: I believe in you adam, but you forgot the aposthrophes

**Comment by ShadowDeath6** [comment]
frankly I think horror talisman should be an AoE used inhand, it'll never be good if it only works on one person since you still slap their ass with it

**Comment by ShadowDeath6** [comment]
come on adam you almost got it

**Comment by utahclock** [comment]
Adam do you even test pr? It fails checks...

File: code/game/gamemodes/cult/talisman.dm [hunk 3]
```diff
@@ -168,6 +169,20 @@
...
+	cultist_desc = "A one-use talisman that will tell your fellow cultists to get around the fucking rune already"
+	color = "#512727" // red-black
+	invocation = "G'et a'round th'e fu'cking ru'ne!"
+	
+/obj/item/weapon/paper/talisman/rune_alert/invoke(mob/living/user, successfuluse = 1)
+	. = ..()
+	
+	for(var/mob/M in mob_list)
+		if(iscultist(M))
+		M.visible_message("<span class='cultlarge'><b>"GET AROUND THE DAMN RUNE ALREADY!"</span>")
```
Comment (ShadowDeath6) [inline code/game/gamemodes/cult/talisman.dm]: indent this

**Comment by Super3222** [comment]
Stop making PR's if you can't test them. Stop using the web editor to make PR's. The process to install toritosegit is as easy as installing byond. If you can't do that, than don't make PR's.

**Comment by ShadowDeath6** [comment]
> toritosegit

i'm crying

**Comment by utahclock** [comment]
What super said cept i use git extension. But i do have tortise installed.

**Comment by Super3222** [comment]
^ A perfect example of what happens when you don't compile the code.

File: code/game/gamemodes/cult/talisman.dm [hunk 4]
```diff
@@ -333,38 +374,76 @@
...
 				user << sound('sound/effects/magic.ogg',0,1,25)
-				qdel(src)
-		if(istype(target, /obj/item/stack/sheet/plasteel))
-			var/quantity = target.amount
-			var/turf/T = get_turf(target)
-			new /obj/item/stack/sheet/runed_metal(T,quantity)
-			target.use(quantity)
+				uses--
+				if(uses <= 0)
+						talisman_consumed()
```
Comment (Tenryumi) [inline code/game/gamemodes/cult/talisman.dm]: Just 1 indention is necessary.

**Comment by X-TheDark** [comment]
IT ONLY TOOK 10 COMMITS TO FIX ALL OF THIS, THAT'S NOT SO BAD, ALL THINGS CONSIDERING, WOULDN'T YOU AGREE???

**Comment by Super3222** [comment]
You're pretty lucky X helped put this PR back together. Otherwise it'd been closed for good.

Since you can't even come onto our server you might as well close this anyway.

**Comment by ghost** [comment]
Being able to play wasn't the reason I coded (and tbqh was more of a downside than anything)

I enjoy making changes and having people respond with accurate, coherent feedback to those changes so they can be edited further. 

The less I have to actually deal with the people on the server the better.

Tl;dr i enjoy coding to see what i think would make the game more fun for other people get in regardless of what I spew in ooc sometimes. I just hate those other people at the moment.

Tl;dr of tl;dr 

I want to code and add new features that people will like while not having to deal with those same people

**Comment by Super3222** [comment]
That's why we're all here, Adam We love making features, and we love hearing coherent feedback. But if you wish to continue to contribute than you're going to have to do 2 things. 1) Download the code. 2) Install a git program. If you need help contact someone on GitHub. 

Otherwise stop coming on here.

**Comment by Super3222** [comment]
DISCORD, not the github.

**Comment by Super3222** [comment]
Contact someone on the discord, NOT the github.

File: code/game/gamemodes/cult/talisman.dm [hunk 5]
```diff
@@ -118,6 +118,7 @@
 	color = "#551A8B" // purple
 	invocation = "Sas'so c'arta forbici!"
 	health_cost = 5
+	uses = 3
```
Comment (Cruix) [inline code/game/gamemodes/cult/talisman.dm:121]: If you are keeping the 3-uses, update the cultist_desc to reflect that. Also think about increasing the health_cost.

**Comment by ShadowDeath6** [comment]
Check if lesser construction talismans ( from archives) can make runed metal please

**Comment by Super3222** [comment]
@AdamElTablawy Update the cultist description already
